lib/Reader/readerir.cpp

```diff
@@ -4739,12 +4774,57 @@ bool GenIR::canMakeDirectCall(ReaderCallTargetData *CallTargetData) {
   return !CallTargetData->isJmp();
 }
 
-IRNode *GenIR::makeDirectCallTargetNode(void *CodeAddr) {
-  uint32_t NumBits = TargetPointerSizeInBits;
-  bool IsSigned = false;
+GlobalVariable *GenIR::GetGlobalVariable(uint64_t LookupHandle,
+                                         uint64_t ValueHandle, Type *Ty,
+                                         StringRef Name, bool IsConstant) {
+  GlobalVariable *GlobalVar = nullptr;
+  auto MapElement = HandleToGlobalVariableMap.find(LookupHandle);
+  if (MapElement != HandleToGlobalVariableMap.end()) {
+    GlobalVar = MapElement->second;
+  }
```

Comment:
I believe you could replace this with`GlobalVariable *&GlobalVar = HandleToGlobalVariableMap[LookupHandle]` and then just assign to GlobalVar below instead of repeating the lookup with operator[] there.